### PR TITLE
Workflows UI updates

### PR DESCRIPTION
**Pull request by longhotsummer**
Workflows in task edit view
Workflow listings
Task selection

### CODE REVIEW

**Comment by longhotsummer** [comment]
5 service level agreements strangely formatted south africa indigo platform 2019-02-08 11-56-17
tester indigo platform 2019-02-08 11-56-01
tester indigo platform 2019-02-08 11-55-52
5 service level agreements strangely formatted indigo platform 2019-02-08 11-55-39

File: indigo_app/views/workflows.py [hunk 1]
```diff
@@ -62,10 +46,10 @@ class WorkflowDetailView(WorkflowViewBase, DetailView):
...
         context = super(WorkflowDetailView, self).get_context_data(**kwargs)
 
-        context['task_groups'] = Task.task_columns(['open', 'pending_review'], self.object.tasks.all())
-
-        # TODO: load from API? filter? exclude closed tasks?
-        context['possible_tasks'] = self.country.tasks.exclude(pk__in=[t.id for t in self.object.tasks.all()]).all()
+        tasks = self.object.tasks.all()
+        context['has_tasks'] = bool(tasks)
+        context['task_groups'] = Task.task_columns(['open', 'pending_review'], tasks)
+        context['possible_tasks'] = self.country.tasks.unclosed().exclude(pk__in=[t.id for t in self.object.tasks.all()]).all()
```
Comment (goose-life) [inline indigo_app/views/workflows.py]: should this not be `place` over `country`?

Comment (longhotsummer) [reply indigo_app/views/workflows.py]: @goose-life I was trying to be a bit cheeky and allow national-level workflows that cover locality tasks. But I realise this also allows you to have a locality workflow that has tasks from other localities, which is a bit weird. I'll push a change.

Comment (longhotsummer) [reply indigo_app/views/workflows.py]: Pushed elsewhere: https://github.com/laws-africa/indigo/pull/353/commits/30edf384422e49895cd4ade56742d324556dc7e0

**Pull request merged**